### ISSUE TITLE
4.3.4 break css 

### ISSUE THREAD

**Issue by rimer10** [orangehrm/orangehrm#584]
php version:PHP 7.2.19-0ubuntu0.19.04.2
mysql:14.14 Distrib 5.7.27, for Linux (x86_64) using  EditLine wrapper

after install successfully my login page look like this 
why thiss css are not loading? 

folder permission : www-data 

![hrm](https://user-images.githubusercontent.com/36982239/67203771-9b6d9f80-f429-11e9-94d4-cb3f80548a77.png)

**Comment by rimer10** [comment]
same think happen i clone from master as well

**Comment by RajithaKumara** [comment]
If you clone, you have to go through [this guide](https://github.com/orangehrm/orangehrm/wiki/Development). Btw previously you tried with [OrangeHRM 4.3.3 release](https://sourceforge.net/projects/orangehrm/files/stable/4.3.3/).

**Comment by rimer10** [comment]
@RajithaKumara i follow[ this guide](https://github.com/orangehrm/orangehrm/wiki/Development) its will fix #514  but this is new issue today also i try what should i do now? 

follow this steps 
1. clone from mater 
2. assign subdomain and do configuration fro apache2 
3 visit config domain 
4. give www-data permission the directory 

after install success check home page UI

**Comment by RajithaKumara** [comment]
Hi @rimer10 
Are there any server side logs? Or browser side console errors? If you can, please check the "Network" tab in browser Developer Tools.

**Comment by rimer10** [comment]
[hr.mydevdemo.xyz](hr.mydevdemo.xyz) this the dev url 

network throw 404 u can also check but why its happen?

**Comment by RajithaKumara** [comment]
Hi @rimer10 
I checked your dev URL. If you can go through 404 URLs in browser **Network** tab, you can see that URLs are request assets from wrong path.
E.g.: 
incorrect URL:- http://domain.name/symfony/web/var/www/html/orangemaster/symfony/web/themes/default/css/main.css
it should be correct as: http://domain.name/symfony/web/themes/default/css/main.css
Additional path appended to URL: `/var/www/html/orangemaster/symfony/web`

Please check your Apache 2.4 `VirtualHost` if configured or check `.htaccess` configurations. If anything should be changed from OrangeHRM opensource, please make pull request.

**Comment by rimer10** [comment]
i checked everything  you can also try maybe u can reproduces this issues 

1.git clone  
2. this is my ornage.conf 
```
<VirtualHost *:80>
    ServerName hr.mydevdemo.xyz
    DocumentRoot /var/www/html/orangemaster
    <Directory /var/www/html/orangemaster>
   Options Indexes FollowSymLinks
        AllowOverride All
        Require all granted
</Directory>
</VirtualHost>

```
3. install 
4.after install success css are missing 

note: .htacess already enable some other website are running (wordpress)

**Comment by RajithaKumara** [comment]
Hi @rimer10 
Did you tried with OrangeHRM 4.3.2? If it worked for you, probably the issue is #548.

**Comment by davidspeijer** [comment]
Same issue here for 4.3.3 did you find a solution?

**Comment by RajithaKumara** [comment]
Hi @davidspeijer 
No issues from OrangeHRM 4.3.3? right?

**Comment by davidspeijer** [comment]
No, at least not in the download at the releases page.

**Comment by RajithaKumara** [comment]
Hi @rimer10 
I found the issue. Did you run `php symfony orangehrm:publish-assets` command within the `<orangehrm-4.3.4>/symfony` directory? I think you missed it while following [the guide](https://github.com/orangehrm/orangehrm/wiki/Development). If it will not solve your issue, please download [OrangeHRM 4.3.4 release](https://sourceforge.net/projects/orangehrm/files/stable/4.3.4/)  and try installing it.

**Comment by sanjeevkpandit** [comment]
> Hi @rimer10
> I found the issue. Did you run `php symfony orangehrm:publish-assets` command within the `<orangehrm-4.3.4>/symfony` directory? I think you missed it while following [the guide](https://github.com/orangehrm/orangehrm/wiki/Development). If it will not solve your issue, please download [OrangeHRM 4.3.4 release](https://sourceforge.net/projects/orangehrm/files/stable/4.3.4/) and try installing it.

This worked for me.

**Comment by RajithaKumara** [comment]
Hi @sanjeevkpandit 
Happy hear your success. :relaxed:

**Comment by kavithagurusamy** [comment]
Fixed in OrangeHRM 4.3.5 version
